y_py.pyi

```diff
@@ -127,10 +127,33 @@ class YDoc:
         If there was an instance with this name, but it was of different type, it will be projected
         onto `YText` instance.
         """
+    def after_transaction_cleanup(
```

Comment:
Would it make sense to prefix this method name with `on_`? I.e. `on_after_transaction_cleanup`. 

It might be fine to rename this to `on_after_transaction`.

Reply:
Sounds good, based on [this comment](https://github.com/y-crdt/y-crdt/pull/120#issuecomment-1121972669) would `observe_after_transaction` be an ideal name?

Reply:
Right, that's even better!